src/olympia/addons/views.py

```diff
@@ -773,6 +774,39 @@ def as_view(cls, **kwargs):
         return non_atomic_requests(view)
 
 
+class AddonAutoCompleteSearchView(AddonSearchView):
+    pagination_class = None
+    serializer_class = ESAddonAutoCompleteSerializer
+
+    def get_queryset(self):
+        # Minimal set of fields from ES that we need to build our results.
+        # It's the opposite tactic used by the regular search endpoint, which
+        # excludes a specific set of fields - because we know that autocomplete
+        # only needs to return very few things.
+        included_fields = (
+            'icon_type',  # Needed for icon_url.
+            'id',  # Needed for... id.
+            'modified',  # Needed for icon_url.
+            'name_translations',  # Needed for... name.
+            'persona',  # Needed for icon_url (sadly).
+            'slug',  # Needed for url.
+            'type',  # Needed to attach the Persona for icon_url (sadly).
```

Comment:
ah, now I understand the persona tests earlier. Maybe they need that same comment or so, I wondered if they belong to a different PR